source/parser.hera

```diff
@@ -5904,6 +5909,43 @@ IndentedJSXChildExpression
 NestedJSXChildExpression
   JSXNested JSXChildExpression
 
+## Using Declaration
+
+UsingDeclaration
+  Using:decl _? UsingBinding:binding ( __ Comma __ UsingBinding )*:tail UsingJSModeError ->
+    const bindings = [binding].concat(tail.map(([,,,b]) => b))
+
+    return {
+      type: "Declaration",
+      children: $0,
```

Comment:
Might be nicer to exclude `UsingJSModeError` here via `[decl, $2, tail]` or maybe name `$2` as `ws`.

Reply:
We need to keep it in the parse tree all the way through to generate.

Reply:
Oh, whoops, of course.